discord.md

```diff
@@ -0,0 +1,30 @@
+/*
+Title: Discord Integration 
+Sort: 3
+*/
+
+Thats right Nightbot is now usable on your very own Discord Server! 
+
+No idea what Discord is? Discord is an all-in-one voice and text chat for gamers that’s free, secure, and works on both your desktop and phone. Check them out at [discordapp.com](https://discordapp.com/)!
+
+## Set-up
+
+1. In order to have Nightbot in your Discord server, you need a Discord Server! If you haven't already, head over to [discordapp.com](https://discordapp.com/) and make yourself a profile and create a server. More info on everything Discord can be found on their [help pages](https://support.discordapp.com/hc/en-us).
+
+2. Log in to Nightbot and head to the [Integrations](https://beta.nightbot.tv/integrations) tab. Here integrations settings for Discord can be found and managed.
+
+3. Clicking Connect will open a pop-out where you can allow Nightbot to access your Discord. (Don't worry we just need your Username and Avatar) Upon granting access you can add a server in which for Nightbot to join. Select a server from the dropdown list and keep the permissions the same. **Note:** Nightbot will **not** kick or ban users from your server.
+
+4. Nightbot should now be in your Discord Server! He has also created a new server role named "Nightbot". (try giving him a color) All of your Commands will be functional in your Discord, so try them out. 
+
+    The next steps are optional but are useful to sync Moderators, Regulars and Subscribers to your Discord and Spam Filtering.
+
+5.  Nightbot has the ability to add roles to users in your stream such as Moderators, Regulars and Subscribers. We understand that Discord as a built in integration to sync subscribers, this is merely to let Nightbot know who the subscribers are. Back in the [Integrations](https://beta.nightbot.tv/integrations) tab open the server settings and select the appropriate server and you are now greeted with assigning user roles. 
+
+6. Select the Role that Nightbot will see moderators in the Discord Server. (Nightbot does not create this by default) You can then to the same for Regulars and Subscribers. To make these roles auto-sync enable "Role Syncing" checkbox. This will enable Nightbot to re-sync and add users every 15 minutes. **Note:** In order to sync for Twitch users must be connected through Twitch in the Integrations Tab in the Discord User Settings.
```

Comment:
> Select the Role that Nightbot will see moderators in the Discord Server.

grammar.

> (Nightbot does not create this by default)

remove

> You can then to the same for Regulars and Subscribers.

reword, perhaps place at the end.

> `and add users`

it doesn't add users..

> In order to sync for Twitch users must be connected through Twitch

we support yt too